src/main/java/com/example/kidz_school/config/properties/ConfirmationTokenConfigProperties.java

```diff
@@ -0,0 +1,12 @@
+package com.example.kidz_school.config.properties;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+@ConfigurationProperties(prefix = "platform.security.confirmation-token")
+public class ConfirmationTokenConfigProperties {
+    private long expiration;
+}
```

Comment:
There is no need of creating a separate class. You can use the @Value annotation